src/pages/patientView/mutation/mock/mutationData.json

```diff
@@ -0,0 +1,150 @@
+[
```

Comment:
Let's explicitly put the endpoint for this data in here: https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/test/mockDataURLs.json. Set the flag `implemented: false`. Once the endpoint is implemented we can generate new mock data and set it to true, so we can make sure the mock data stays in sync.